md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/UserRolesAPI.java

```diff
@@ -181,4 +191,27 @@ public Object execute(String[] params) {
     }
 
 
+    @RequestMapping(value = {"/options", "/options/"}, method = RequestMethod.POST)
+    @ResponseBody public
+    RestWrapperOptions listOptions(Principal principal) {
+        RestWrapperOptions restWrapperOptions = null;
+        try {
+            Map<Integer,String> objects=userRolesDAO.diffRoleList();
+            Iterator it = objects.entrySet().iterator();
+            List<RestWrapperOptions.Option> options = new ArrayList<RestWrapperOptions.Option>();
+            while (it.hasNext()) {
+                Map.Entry pair = (Map.Entry)it.next();
+                System.out.println(pair.getKey() + " = " + pair.getValue());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)